css-basics/social-card/css/styles.css

```diff
@@ -1,3 +1,43 @@
 * {
   box-sizing: border-box;
 }
+
+body {
+  background-color: lightgray;
+  font-family: Arial, sans-serif;
+}
+
+article {
+  background-color: white;
+  border-radius: 8px;
+  padding: 20px;
+}
+
+img {
+  width: 100px;
+  border-radius: 8px;
+}
+
+h2 {
+  text-transform: uppercase;
+}
+
+h2,
+p {
+  margin: 12px 0;
+}
+
+p.account {
+  color: gray;
+}
+
+button {
```

Comment:
Fantastic work as was to be expected - only thing I could find: Button seems to be a little bigger in the original, with darker blue and slightly different arrow (not that I would have solved this...)